config/kubernetes/sig-scheduling/teams.yaml

```diff
@@ -2,170 +2,82 @@ teams:
   sig-scheduling-api-reviews:
```

Comment:
api-reviewers? It's actually just me https://github.com/kubernetes/kubernetes/blob/7b9a20b587de81ed7b885e97f2fd1db1deca548b/OWNERS_ALIASES#L452

Reply:
This would be just for Github notification purposes, not technically "assigned" to someone. So should be fine IMO.

Reply:
If appliable, plz add me to the notify list.

Reply:
+1 I am interested in it.